index.js

```diff
@@ -142,8 +142,22 @@ class ClientServiceBase {
     return this._executePromise(promise);
   }
 
+	/**
+	 *
+	 * @returns {Promise<CertificateRegistrationRequest>}
+	 */
+	async buildCertificateRegistrationRequest() {
+  	const builder = new CertificateRegistrationRequestBuilder(
```

Comment:
How about i just reuse the `createSerializedFunctionRegistrationRequest()` code in the `registerCertificate()`, and it will become like this
```
  async registerCertificate() {
   const request = await this.createSerializedCertificateRegistrationRequest();
    const promise = new Promise((resolve, reject) => {
      this.ledgerPrivileged.registerCert(
          request,
          this.metadata,
          (err, _) => {
            if (err) {
              reject(err);
            } else {
              resolve();
            }
          },
      );
    });

    return this._executePromise(promise);
  }
```

Reply:
I don't think we need to serialize when  not returning serialized byte array, so I think something like below should be the way.

```
registerCertificate() {
  request = createCertificateRegistrationRequest();
  // ...
}

createSerializedCertificateRegistrationRequest() {
  request = createCertificateRegistrationRequest();
  return request.serializedBinary();
}
```